frontend/package.json

```diff
@@ -53,7 +53,7 @@
     "@babel/preset-typescript": "^7.16.0",
     "@blueprintjs/eslint-plugin": "^2.2.0",
     "@next/eslint-plugin-next": "^11.1.2",
-    "@playwright/test": "^1.27.0",
+    "@playwright/test": "^1.32.3",
```

Comment:
upgrade Playwright to use `page.toBeInViewport()`

Reply:
Excellent